### PR TITLE
Add Exokit issue templates

### PR DESCRIPTION
**Pull request by chrislatorres**
This PR adds issue templates for
- General Exokit Desktop Bugs
- Exokit Magic Leap Bugs

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/exoit-magic-leap-bug-report.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+name: Exoit Magic Leap Bug report
+about: File an issue to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
```
Comment (modulesio) [inline .github/ISSUE_TEMPLATE/exoit-magic-leap-bug-report.md]: Maybe more practically phrased as “what happened, and what did you expect to happen”?

File: .github/ISSUE_TEMPLATE/exoit-magic-leap-bug-report.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+3. Scroll down to '....'
+4. See error
+
+**Additional context**
+Add any other context about the problem here.
+
+**System information:**
+ - OS: [e.g. Windows 10]
+ - Exokit Version [e.g. v0.0.512]
+-  Node/other dependencies versions
```
Comment (modulesio) [inline .github/ISSUE_TEMPLATE/exoit-magic-leap-bug-report.md]: I think not important for ml case.

Comment (modulesio) [reply .github/ISSUE_TEMPLATE/exoit-magic-leap-bug-report.md]: Url is however important.

File: .github/ISSUE_TEMPLATE/exokit-desktop-bug-report.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+3. Scroll down to '....'
+4. See error
+
+**Additional context**
+Add any other context about the problem here.
+
+**System information:**
+ - OS: [e.g. Windows 10]
+ - Exokit Version [e.g. v0.0.512]
+-  Node/other dependencies versions
```
Comment (modulesio) [inline .github/ISSUE_TEMPLATE/exokit-desktop-bug-report.md]: Node and npm versions. Npm build log if applicable. Also, downloaded vs built.

Comment (chrislatorres) [reply .github/ISSUE_TEMPLATE/exokit-desktop-bug-report.md]: 2494818

File: .github/ISSUE_TEMPLATE/exokit-desktop-bug-report.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+name: Exokit Desktop Bug report
+about: File an issue to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
```
Comment (modulesio) [inline .github/ISSUE_TEMPLATE/exokit-desktop-bug-report.md]: Same as the other issue template?

File: .github/ISSUE_TEMPLATE/exokit-desktop-bug-report.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+3. Scroll down to '....'
+4. See error
+
+**Additional context**
+Add any other context about the problem here.
+
+**System information:**
+ - OS: [e.g. Windows 10]
+ - Exokit Version [e.g. v0.0.512]
+ - Node/other dependencies versions
```
Comment (modulesio) [inline .github/ISSUE_TEMPLATE/exokit-desktop-bug-report.md]: What's "other"?

Comment (chrislatorres) [reply .github/ISSUE_TEMPLATE/exokit-desktop-bug-report.md]: Removed it, having "other" is too general for being in a template

File: .github/ISSUE_TEMPLATE/exokit-desktop-bug-report.md [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+**Additional context**
+Add any other context about the problem here.
+
+**System information:**
+ - OS: [e.g. Windows 10]
+ - Exokit Version [e.g. v0.0.512]
+ - Node version
+ - Npm version/build log if applicable
+ - Is your Exokit downloaded or built?
+
```
Comment (modulesio) [inline .github/ISSUE_TEMPLATE/exokit-desktop-bug-report.md]: Kinda inconsistent spacing here. Was this intentional?

Comment (chrislatorres) [reply .github/ISSUE_TEMPLATE/exokit-desktop-bug-report.md]: fixed in 815afce

**Pull request merged**